tests/unit/plugins/test_autotools.py

```diff
@@ -41,7 +43,16 @@ class Options:
             artifacts = []
 
         self.options = Options()
-        self.project_options = snapcraft.ProjectOptions()
+        self.project = snapcraft.project.Project(
+            snapcraft_yaml_file_path=self.make_snapcraft_yaml(
+                textwrap.dedent(
+                    """\
+                    name: make-snap
```

Comment:
oh bummer, maybe a generic name should have been chosen for the copy pasta to be easier :-)

Reply:
Grrr.... I knew that was a dangerous pattern to start. `test-snap` it is :stuck_out_tongue: .